README.md

```diff
@@ -65,33 +65,33 @@ settings page of your project on GitHub. See [Creating & using secrets].
 ### Publishing with OpenID Connect
```

Comment:
@woodruffw do we keep the OIDC mention in this title? Is it for searchability with `Ctrl+F`?

Reply:
Whoops, I missed it with a search. That should probably be "Publishing with trusted publishers" or "Trusted publishing" or similar, thoughts?

Reply:
I'd go for the latter since the former implies there's many publishers that the action supports while the reality is that it's tightly coupled with the GitHub platform, hence there's only one possible publisher — GitHub Actions CI/CD workflows, eh?

Also, while on it, maybe add a sentence/note somewhere like "Trusted publishing is sometimes referenced by it's underlying technology — OIDC or OpenID Connect". This would make sure that people can find the docs using those search terms they might've heard elsewhere...

Reply:
Sounds good, and good idea! Will do in a moment.

Reply:
03a45ac

Reply:
Thanks! Marking as resolved.